Data/Uniques/staff.lua

```diff
@@ -130,9 +130,20 @@ Gain (10-20)% of Elemental Damage as Extra Chaos Damage
 {variant:3}(120-150)% increased Spell Damage if you've dealt a Critical Strike Recently
 ]],[[
 Dying Breath
+Coiled Staff
```

Comment:
this should probably be marked as an old variant (see https://pathofexile.gamepedia.com/Dying_Breath)

Reply:
It's a base change. Variants don't support switching between bases. All other items with base changes are implemented like this in PoB. On the Coiled Staff base, the implicit has a range, on the Iron staff base, it has not. Which means you need to add two bases.

Reply:
Two items is okay because of the bases, but the old one should be marked as an old variant. You can look at 'The Searing Touch' which also has had a base change and the discontinued one is marked as 'Variant: Pre 2.0.0' and the new as 'Variant: Current'.